packages/frontend/src/model/periods.ts

```diff
@@ -488,6 +490,41 @@ export const useExportPraise = (): useExportPraiseReturn => {
   return { exportPraise };
 };
 
+type useExportSummaryPraiseReturn = {
+  exportSummaryPraise: (
+    period: PeriodDetailsDto,
+    exportContext: string,
+    supportPercentage: boolean
+  ) => Promise<Blob | undefined>;
+};
+
+/**
+ * Returns function that exports all praise in a period as csv data.
+ */
+export const useExportSummaryPraise = (): useExportSummaryPraiseReturn => {
+  const apiAuthClient = useApiAuthClient();
+
+  const exportSummaryPraise = async (
+    period: PeriodDetailsDto,
+    exportContext: string,
+    supportPercentage: boolean
+  ): Promise<Blob | undefined> => {
+    if (!period) return undefined;
+
+    const response = await apiAuthClient.get(
+      `/admin/periods/${period._id}/exportSummary?context=${exportContext}&supportPercentage=${supportPercentage}`,
```

Comment:
`/exportSummary` returns summarised praise data for the period, basically the input to the custom export
New endpoint: `/customExport` does what `/exportSummary` currently does.